ext/dom/tests/modern/spec/NamedNodeMap_dimensions.phpt

```diff
@@ -22,7 +22,7 @@ foreach ($test_values as $value) {
 ?>
 --EXPECTF--
 --- -1 ---
-string(1) "a"
+string(3) "N/A"
```

Comment:
The refactor actually fixed a bug here

Reply:
I would not backport this as this is a very invasive fix.